### DISCUSSION TITLE
Concerns in SAF Domain descriptions  to be removed?

### DISCUSSION THREAD

**Discussion by haarer** [GfSE/SAF-Specification#33] [Q&A]
Currently the Domains have concern descriptions which are partly redundant with the concerns of viewpoints, see below for an example.

We should remove them, at least don't call them concerns. The purpose/objective should be sufficient.

[Functional Domain Concerns](https://github.com/GfSE/SAF-Specification/blob/main/README.md#functional-domain-concerns)

- Defining the System Boundary
  - Identification of interaction partners and system interfaces
- Consolidating Functional Requirements:
  - formally specifying the requirements of the system behavior using a black box approach
- Mastering functional dependency:
  - detection and resolution of inconsistencies within the Functional Requirements (known as feature interaction)
- Reducing functional complexity:
   - structuring the functionality from the System's point of view
 - Understanding functional interrelationships:
    - collecting and analyzing the exchange between different functionalities

**Comment by haarer** [comment]
christian hat's erledigt :+1:

**Reply by clalitsc** [reply]
Die Aussage ist nur teilweise zutreffend. "Gelöscht" wurden die Domain Concerns auf der GitHub SAF Spezifikationsseite, main branch, vgl. https://github.com/GfSE/SAF-Specification/
Nicht "Gelöscht" wurden die Domain Concerns im SAF_Cameo_Profile Modell, dort sind diese weiterhin in den Beschreibungstexten der Grid Domain Attribute O-F-L-P zu finden. Und gerade dort stören mich die Concern Beschreibung, da diese eine Inkonsistenz zu dem SAF VP-Concern-Rational-VP.Stakeholder Ansatz darstellen. Wer nun Wert auf diese Concerns legt, möge diese zeitnah zu VP-Concern-Rational-VP.Stakeholder umwandeln.
@GfSE/saf

**Reply by haarer** [reply]
Jetzt ist es auch aus der Grid Domain raus.
